libs/login/build.gradle

```diff
@@ -18,7 +18,6 @@ android {
         buildConfig true
     }
     lint {
-        enable += 'UnknownNullness'
         disable 'UseTomlInstead'
```

Comment:
Suggestion (💡): I understand that we could remove this here too, now that all dependencies within this module are migrated to version catalogs. And then get rid of the whole `lint { ... }` block, just like you did for the rest of the modules.

PS: We could potentially do the same for the `fluxc` module, but we would first need to migrate the below two dependencies to version catalogs (however, I am not sure about the `IgnoreWithoutReason`):

```groovy
    api "com.goterl:lazysodium-android:5.0.2@aar"
    api "net.java.dev.jna:jna:5.15.0@aar"
```

Reply:
> PS: We could potentially do the same for the fluxc module, but we would first need to migrate the below two dependencies to version catalogs

This is not easy to do though, because of the aar specification not working as intended. This is needed, because both of these dependencies offer both `aar` and `jar` and one of them even defaults to `jar`. This seems to be a known issue at Gradle (https://github.com/gradle/gradle/issues/21267) but not yet fixed.

But regarding the `login` module, I agree, I'll fix this in a sec!

Reply:
This looks neat, but I'm unable to verify that `aar` instead of `jar` is actually used. 

Let's take the JNA dependency for the example: it offers both `aar` and `jar`. The default artifact specified in the pom file is `jar`: https://mvnrepository.com/artifact/net.java.dev.jna/jna/5.15.0

When I applied your suggestion and when I run `:WordPress:dependencyInsight --dependency net.java.dev.jna:jna --configuration wordpressWasabiDebugRuntimeClasspath`, it seems that `jar` is used. You can compare this result to result of any other dependency, e.g. `:WordPress:dependencyInsight --dependency androidx.credentials:credentials --configuration wordpressWasabiDebugRuntimeClasspath` which prints usage of `aar`.

But what's strange, when I run the dependency insight for JNA on `trunk`, I also see that `jar` is requested 😕 .

For this reason, as I'm not entirely sure how it all work, I'd postpone changing this dependencies to version catalog as I'm not sure what would be the effect. Happy to hear your thoughts - maybe there's a way to make sure `aar` is used?

Reply:
@wzieba , I did a simple `println(...)` to verify the exact string that is used. In the end, using this `.get().toString()` trick should give us exactly what we had before, a simple string, there shouldn't be any more magic involved, not having version catalogs choosing between `aar` and `jar` whatsoever (🤔). This is my assumption at least. 🤞 

```groovy
println("LazySodium: ${libs.terl.lazysodium.android.get().toString()}")
println("JNA: ${libs.jna.get().toString()}")
```

Reply:
Gotcha 👍 when I removed `@aar` the project won't compile, and with the code you suggested, build is complete so I think we're all good 👍 

I've also applied changed to remove `IgnoreWithoutReason` from disabled rules.